sdk/web-pubsub/web-pubsub-client-protobuf/src/clientProto.js

```diff
@@ -0,0 +1,4126 @@
+/*eslint-disable block-scoped-var, id-length, no-control-regex, no-magic-numbers, no-prototype-builtins, no-redeclare, no-shadow, no-var, sort-vars*/
```

Comment:
if you put it in generated I think it will get ignored from lint

Reply:
This part of `eslint-disable` is totally generated. So I just put it as-is